bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/PartDEventRow.java

```diff
@@ -1,163 +1,238 @@
 package gov.hhs.cms.bluebutton.datapipeline.rif.model;
 
+import java.math.BigDecimal;
 import java.time.LocalDate;
+import java.util.Optional;
 
 /**
- * Models rows from <code>PDE</code> RIF Files.
+ * <p>
+ * Models rows from {@link RifFileType#PDE} RIF Files.
+ * </p>
+ * <p>
+ * The RIF file layout used here is specific to the Blue Button API's ETL
+ * process. The layouts of these files is detailed in the
+ * <code>bluebutton-data-pipeline-rif/dev/rif-record-layout.xlsx</code> file.
+ * The columns contained in the files are largely similar to those detailed in
+ * <a href="https://www.ccwdata.org/web/guest/data-dictionaries">CCW: Data
+ * Dictionaries</a>.
+ * </p>
+ * 
+ * <p>
+ * Design Note: This class is too painful to maintain as a bean, so I've
+ * stripped it down to just a struct. To be clear, this is <strong>not</strong>
+ * immutable and thus <strong>not</strong> thread-safe (and it really shouldn't
+ * need to be).
+ * </p>
  */
 public class PartDEventRow {
-	public static int COMPOUND_CODE_NOT_COMPOUNDED = 1;
-	public static int COMPOUND_CODE_COMPOUNDED = 2;
+	public static final int COMPOUND_CODE_NOT_COMPOUNDED = 1;
+	public static final int COMPOUND_CODE_COMPOUNDED = 2;
+
+	public static final Character DRUG_CVRD_STUS_CD_COVERED = new Character('C');
+	public static final Character DRUG_CVRD_STUS_CD_SUPPLEMENT = new Character('E');
+	public static final Character DRUG_CVRD_STUS_CD_OTC = new Character('O');
+
```

Comment:
In general, I recommend enums for these kinds of constants, as you can add a `static parseRifValue(...)` method to them. I think that ends up making the code a bit more readable and maintainable, but YMMV. Just a suggestion.